### ISSUE TITLE
Multiple maintainers on the vscode app store

### ISSUE THREAD

**Issue by alanz** [alanz/vscode-hie-server#55]
At the moment I am the only one able to publish this package when it gets updated.

Does anyone know how we can add some other people too? I was thinking @Tehnix , @halhenke and @AlexeyRaga to start with, based on the contributors page.

**Comment by alanz** [comment]
And it looks like I can add people with defined roles here https://marketplace.visualstudio.com/manage/publishers/alanz

**Comment by Tehnix** [comment]
Alternatively we could, as I mentioned in https://github.com/alanz/vscode-hie-server/pull/28, set up travis to publish the extensions on either pushes to master branches or tags?

Adding people is also fine with me :)

**Comment by alanz** [comment]
Maybe we should do both?

**Comment by Tehnix** [comment]
I think you only need to do step 3. in the PR then,

3) Go to the travis page for the project at, https://travis-ci.org/alanz/vscode-hie-server
   a) Under `More Options` go into `Settings`
   b) Set a environment variable `VS_TOKEN` with the value of your VS Code personal token
   c) Make sure `Display value in build log` is turned OFF!

and then we cut a tagged release, which should initiate the deploy part of the script. It's the same as I've personally used on my own extension, so it should work immediately.

**Comment by alanz** [comment]
@Tehnix 

Ok, I agree on doing both.  I will get going on the travis side, can you give me whatever details I need to add you to the project?

**Comment by alanz** [comment]
Stupid question, how do I get my hands on my `VS Code personal token`?

**Comment by alanz** [comment]
@Tehnix I finally dug through the rabbit holes enough and created a token, attached to https://alan-zimm.visualstudio.com

I set it to be limited to markeplace and publish to marketplace, and made it an environment variable on travis.

Next step is yours.

**Comment by Tehnix** [comment]
Not sure which would you need, but my publisher name is `tehnix` and my email `christianlaustsen@gmail.com`.

~~Currently there's an issue with it, if I remember correctly, the VS_TOKEN environment variable you set in travis needs to be with `"` around the value.~~ Nevermind, the env variable was called `VS_CODE`, fixed it in the script :)

To deploy to the marketplace, you now simply have to

1. bump the version number in `package.json` and,
2. cut a release/tag, 

such as with,

```
$ git tag -a 0.0.9 -m "Version 0.0.9"
$ git push origin 0.0.9
```

(or as I just did `git commit -am "Bump version number to 0.0.9" && git push && git tag -a 0.0.9 -m "Version 0.0.9" && git push origin 0.0.9`, there's probably a shorter way...)

It then starts to build the master branch that was just tagged, which you can follow along on [the travis-ci build](https://travis-ci.org/alanz/vscode-hie-server/jobs/347993608) (after I finally figured out the problem), and et voilà ! it's now live [on the marketplace]( https://marketplace.visualstudio.com/items?itemName=alanz.vscode-hie-server) :)

I suspect it'll also work when creating a [GitHub release](https://help.github.com/articles/creating-releases/), which should create a tag in the process.

**Comment by alanz** [comment]
I just got a mail from the marketplace saying 0.0.9 validated and was
published. Looks like.it works.

Will add the user in the morning.

**Comment by alanz** [comment]
@Tehnix help

The personal access token has expired, I have created a new one, but it does not work.

Any pointers?

cc @jneira

**Comment by Tehnix** [comment]
@alanz hmm, my first instinct would be that it might be the permission scopes? I've found the access token management super non-intuitive.

It should look something like the following:

![Screenshot 2020-05-06 at 23 21 39](https://user-images.githubusercontent.com/1189998/81229645-71a84e80-8ff0-11ea-9663-97b4dfefaf9d.png)

Note the scope of `Marketplace (Acquire)` + `Marketplace (Manage)` along with `All accessible organizations`.

**Comment by alanz** [comment]
ok, I have that, but not "All Accessible Organizations", will try changing that

**Comment by Tehnix** [comment]
At least they highlight it at https://code.visualstudio.com/api/working-with-extensions/publishing-extension#get-a-personal-access-token so I assume it's important. I think they changed how they set up the tokens. It also only lasts a year now, so that's something to keep in mind.

**Comment by alanz** [comment]
yes, that worked, at last. Hallelujah. I was following that guide, but missed that bit.  You end up with bits of info in about three different pages, and finding the right, current one is a pain.
